physics/ephemeris_test.cpp

```diff
@@ -499,7 +499,7 @@ TEST_P(EphemerisTest, EarthProbe) {
   // The solution is a line, so the rounding errors dominate.  Different
   // compilers result in different errors and thus different numbers of steps.
```

Comment:
Change this to "different libms" as we now know the compilers are innocent (unless we get the flags wrong). The above involves `std::pow` to adapt the step size. I am unsure why `earth_positions` has variable errors, but there must be something similar somewhere...